### PR TITLE
v1.11: Language settings

### LINKED ISSUE #2985: v1.11: Localization setting and parameter now also accept ISO-639-1 bigrams
v1.10 introduce a new localization index setting and a new localization search parameter. These settings accept one or more of a subset of ISO-639-3 trigram locales. v1.11 expands functionality by allowing users to use ISO-639-1 bigrams.

## Tasks
- [ ] Update `/reference/api/settings#localized-attributes-object`
- [ ] Update `/reference/api/search#query-locales`

## References
- Engine issue: https://github.com/meilisearch/meilisearch/issues/4827
- PRD: none (?)
- SME: @ManyTheFish

### PR DESCRIPTION
**Pull request by guimachiavelli**
Closes #2985

### CODE REVIEW

File: .code-samples.meilisearch.yaml [hunk 1]
```diff
@@ -1373,7 +1373,7 @@ search_parameter_reference_locales_1: |-
   -X POST 'http://localhost:7700/indexes/INDEX_NAME/search' \
   -H 'Content-Type: application/json' \
   --data-binary '{
-    "q": "進撃の巨人",
+    "q": "QUERY TEXT IN JAPANESE",
```
Comment (curquiza) [inline .code-samples.meilisearch.yaml:1376]: why do we change? what was the issue with the previous example?

Comment (guimachiavelli) [reply .code-samples.meilisearch.yaml:1376]: To make this sample consistent with the pattern we've been following with new code samples, where `UPPERCASE TEXT` signals placeholder content. A bit like the URL a couple lines above: `http://localhost:7700/indexes/INDEX_NAME/search`.

**Review by curquiza — APPROVED** [review]

**Comment by guimachiavelli** [comment]
pinging @ManyTheFish for review

**Review by ManyTheFish — APPROVED** [review]
Hey @guimachiavelli, LGTM, sorry again for the delay!

**Pull request merged**